sdk/playwrighttesting/microsoft-playwright-testing/src/utils/utils.ts

```diff
@@ -76,6 +97,9 @@ export const validateMptPAT = (): void => {
     if (Date.now() >= claims.exp! * 1000) {
       exitWithFailureMessage(ServiceErrorMessageConstants.EXPIRED_MPT_PAT_ERROR);
```

Comment:
exitWithFailureMessage will terminate the reporter process as well, all of these should be done via the handleFailure callback